go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	github.com/libp2p/go-tcp-transport v0.2.1
 	github.com/multiformats/go-multiaddr v0.3.1
 	github.com/onflow/cadence v0.10.5
-	github.com/onflow/flow-core-contracts/lib/go/contracts v0.3.1-0.20201122012505-4061d358b8db
+	github.com/onflow/flow-core-contracts/lib/go/contracts v0.6.1-0.20210113125452-03c054c55935 // https://github.com/onflow/flow-core-contracts/pull/96 -> https://github.com/onflow/flow-core-contracts/tree/03c054c55935bf8a1963933104c72773c4074e69
```

Comment:
We should merge the core contracts PR and tag a new version for it, then specify that version here, before merging this PR

Reply:
there is a slight cyclical dependency going on there. If I merge flow-core-contracts it will have broken tests because i need to update the emulator first.

The plan is: 
- I update the emulator
- fix the emulator version to a branch inside flow-core-contracts to make the tests pass
- merge and tag flow-core-contracts
- update flow-core-contracts version in flow-go
- merge and tag flow-go
- update flow-go version in flow-emulator
- merge and tag flow-emulator
- finally  update flow-emulator version in  flow-core-contracts and update the tag

Reply:
That makes sense. Sounds good!